va/validation-authority_test.go

```diff
@@ -308,18 +306,6 @@ func TestSimpleHttp(t *testing.T) {
 	test.AssertEquals(t, invalidChall.Error.Type, core.UnknownHostProblem)
 	va.TestMode = true
 
-	chall.Path = pathUnsafe
-	invalidChall, err = va.validateSimpleHTTP(ident, chall)
-	test.AssertEquals(t, invalidChall.Status, core.StatusInvalid)
-	test.AssertError(t, err, "Path doesn't consist of URL-safe characters.")
-	test.AssertEquals(t, invalidChall.Error.Type, core.MalformedProblem)
-
-	chall.Path = pathUnsafe302
-	invalidChall, err = va.validateSimpleHTTP(ident, chall)
-	test.AssertEquals(t, invalidChall.Status, core.StatusInvalid)
-	test.AssertError(t, err, "Redirect should have failed.")
-	test.AssertEquals(t, invalidChall.Error.Type, core.ConnectionProblem)
-
```

Comment:
Agreed on both points: These test cases seem useful, we should keep them. And we should try to log IP addresses for each HTTP request performed.